learningresources/models.py

```diff
@@ -1,48 +1,48 @@
 """
-Learning objects data model
+Learning resources data model
 """
+
 from django.db import models
 from django.contrib.auth.models import User
 
 
 class Course(models.Model):
-
     """
     A course on edX platform (MITx or residential).
     """
     repository = models.ForeignKey('Repository')
     org = models.TextField()
-    course_number = models.IntegerField()
+    course_number = models.TextField()
     semester = models.TextField()
-    import_date = models.DateField()
+    import_date = models.DateField(auto_now_add=True)
     imported_by = models.ForeignKey(User)
 
 
-class LearningObject(models.Model):
+class LearningResource(models.Model):
     """
     The units that compose an edX course:
     chapter, sequential, vertical, problem, video, html, etc.
     """
     course = models.ForeignKey(Course)
-    learning_object_type = models.ForeignKey('LearningObjectType')
+    learning_resource_type = models.ForeignKey('LearningResourceType')
     uuid = models.TextField()
     title = models.TextField()
     description = models.TextField()
     content_xml = models.TextField()
     path_xml = models.TextField()
```

Comment:
This is an unused field, do we have plans for it?  It looks like we are storing etree path in in mpath, so wouldn't that make this redundant anyway?